eu.ngdevelopment.tsukitag.json

```diff
@@ -0,0 +1,77 @@
+                "mkdir /app/bin",
+                "tar -xzf tsukitag.tar.gz -C /app/bin",
+                "install -Dm644 eu.ngdevelopment.tsukitag.appdata.xml /app/share/metainfo/eu.ngdevelopment.tsukitag.appdata.xml",
+                "install -Dm644 eu.ngdevelopment.tsukitag.desktop /app/share/applications/eu.ngdevelopment.tsukitag.desktop",
+                "install -Dm644 256x256.png /app/share/icons/hicolor/256x256/apps/eu.ngdevelopment.tsukitag.png",
+                "install -Dm644 128x128.png /app/share/icons/hicolor/128x128/apps/eu.ngdevelopment.tsukitag.png",
+                "install -Dm644 64x64.png /app/share/icons/hicolor/64x64/apps/eu.ngdevelopment.tsukitag.png",
+                "install -Dm644 32x32.png /app/share/icons/hicolor/32x32/apps/eu.ngdevelopment.tsukitag.png",
+                "install -Dm644 16x16.png /app/share/icons/hicolor/16x16/apps/eu.ngdevelopment.tsukitag.png"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://github.com/ngelsi/tsuki-tag-native/releases/download/v.0.9.2/tsukitag-linux-x64-uncompressed-v092.tar.gz",
+                    "sha256": "d9e0f22bb27a2293fdf3634b3f98631a7bc0f0227204a972c2414e1162230438",
+                    "dest-filename": "tsukitag.tar.gz"
+                },
+                {
+                    "type": "file",
+                    "path": "eu.ngdevelopment.tsukitag.desktop",
+                    "dest-filename": "eu.ngdevelopment.tsukitag.desktop"                    
+                },
+                {
+                    "type": "file",
+                    "path": "eu.ngdevelopment.tsukitag.appdata.xml",
+                    "dest-filename": "eu.ngdevelopment.tsukitag.appdata.xml"                    
+                },                
+                {
+                    "type": "file",
+                    "path": "./icons/256x256.png",
+                    "dest-filename": "256x256.png"
+                },
+                {
+                    "type": "file",
+                    "path": "./icons/128x128.png",
+                    "dest-filename": "128x128.png"
+                },
+                {
+                    "type": "file",
+                    "path": "./icons/64x64.png",
+                    "dest-filename": "64x64.png"
+                },
+                {
+                    "type": "file",
+                    "path": "./icons/32x32.png",
+                    "dest-filename": "32x32.png"
+                },
+                {
+                    "type": "file",
+                    "path": "./icons/16x16.png",
+                    "dest-filename": "16x16.png"
+                }                
+            ]
+        }
+    ],
+    "finish-args": [
+        "--filesystem=xdg-pictures",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=network"
+    ],
+    "cleanup": [
+        "rm tsukitag.tar.gz"
```

Comment:
1. cleanup isn't a list of command
2. cleanup is to clean installed files not downloads or build artifacts